src/classes/chat_registry.py

```diff
@@ -234,6 +261,20 @@ def _store_club(self, club: Club) -> None:
             with open(file_path, "w") as file:
                 json.dump(club_data, file)
 
+    def _update_club(self, club: Club) -> None:
```

Comment:
We don't plan to append new data, just overwrite it. Maybe then we should remove one of the methods due to their similarity? `_store_club` `_update_club`